build_tools/autogen_ltc_backend.yaml

```diff
@@ -23,6 +24,7 @@ blacklist:
 - arange
 - arange.start
 - arange.start_step
+- fill.Scalar
```

Comment:
@antoniojkim, after adding support for `aten.fill.Scalar` in torch-mlir, some of the tests that use the op `aten.fill_.Scalar` (note the underscore), started to fail with the error:

```
        Runtime error: Traceback (most recent call last):
          File "/usr/local/google/home/ramiroleal/torch-mlir/build/tools/torch-mlir/python_packages/torch_mlir/torch_mlir_e2e_test/framework.py", line 302, in compile_and_run_test
            trace = config.run(compiled, golden_trace)
          File "/usr/local/google/home/ramiroleal/torch-mlir/build/tools/torch-mlir/python_packages/torch_mlir/torch_mlir_e2e_test/configs/lazy_tensor_core.py", line 35, in run
            output = getattr(artifact, item.symbol)(*lazy_inputs)
          File "/usr/local/google/home/ramiroleal/torch-mlir/build/tools/torch-mlir/python_packages/torch_mlir/torch_mlir_e2e_test/test_suite/constant_alloc.py", line 364, in forward
            memory_format=torch.preserve_format).fill_(0)
        RuntimeError: Unimplemented Error: 
        	/usr/local/google/home/ramiroleal/torch-mlir/build/tools/torch-mlir/python/torch_mlir/csrc/base_lazy_backend/generated/shape_inference.cpp:35    std::vector<torch::lazy::Shape> torch::lazy::compute_shape_fill(const at::Tensor &, const at::Scalar &)
```

My guess is that LTC was converting the ops to `aten.fill.Scalar`, causing issues. I noticed that `aten.fill.Scalar` is currently not supported in the [TS backend](https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/ts_native_functions.yaml), so I added the op to this list for consistency. This fixed the test failures.

Let me know if this change is right or if something else needs to be modified.

Reply:
Yep, the change looks right to me